src/components/Footer/Footer.css

```diff
@@ -0,0 +1,76 @@
+.footer {
+    position: relative; /* Ensure positioning for pseudo-element */
+    display: flex;
+    justify-content: space-between;
+    padding: 20px;
+    background-color: #f8f8f8;
+    border-top: 1px solid #e7e7e7;
+}
+
+/* Add light blue strip at the top of footer */
+.footer::before {
+    content: '';
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%; /* Full width of the footer */
+    height: 4px; /* Height of the strip */
+    background-color: #5AB2FF; /* Light blue color */
+}
+
+/* Adjusted footer layout */
+.footer-about {
+    flex: 1;
+    margin-right: 20px;
+    margin-left: 50px;
+    font-size: 14px;

```

Comment:
Do not use font size in `px`, instead user `rem`. This helps to keep the text in the web[page responsive to different screen sizes. Use the font sizes defined in `theme.tsx`.